alpha-blog/app/views/articles/show.html.erb

```diff
@@ -1,16 +1,23 @@
-<%= render 'messages' %>
-<h1>Showing articles details</h1>
-
-<p>
-  <b>Title: </b>
-  <%= @article.title %>
-</p>
-<p>
-  <b>Description: </b>
-  <%= @article.description %>
-</p>
-
-<%= link_to 'Edit', edit_article_path(@article) %> |
-<%= link_to 'Delete', article_path(@article), method: :delete, data: { confirm: "Are you sure?"} %> |
-<%= link_to 'Return to articles listing', articles_path %>
+<div id="page-content">
+  <h2 class="text-center mt-4"><%= @article.title %></h2>
+  <div class="container">
+    <div class="row justify-content-md-center">
+      <div class="col-8 mt-4">
+        <div class="card text-center shadow mb-5 bg-white rounded">
+          <div class="card-header font-italic">
+            by Lee Ren Jie
+          </div>
+          <div class="card-body">
+            <div class="card-text text-start"><%= simple_format(@article.description) %></div>
+            <a href="#"><%= link_to 'Edit', edit_article_path(@article), class: "btn btn-outline-primary" %></a>
```

Comment:
Why `link_to` within `a` tag? you inspec element here and you'll see a tag within a tag